build/dependencies.props

```diff
@@ -2,85 +2,30 @@
   <!-- These package version may be overridden or updated by automation. -->
   <PropertyGroup Label="Package Versions: Auto" Condition=" '$(DotNetPackageVersionPropsPath)' == '' ">
     <MicrosoftCSharpPackageVersion>4.5.0-preview2-26403-05</MicrosoftCSharpPackageVersion>
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0-preview2-26403-06</MicrosoftDotNetPlatformAbstractionsPackageVersion>
```

Comment:
Nice to get rid of all of these! Do we have a "safe" way of knowing whether this will pass? Can we make sure a local build of Universe is done to mitigate potentially breaking the CI?

Reply:
We could enable PR checks that build Universe, but otherwise, not really. I feel pretty okay about this though because I did a search over all source code to make sure packages I've removed are not referenced. If this fails, it's easy to revert.

Reply:
Yeah, this could really save us from thoughtlessly taking dependencies in new places just because  we think it's already in use.

Reply:
Yeah that's fine.